docs/markdown/guides/mermaid/index.ts

```diff
@@ -0,0 +1,62 @@
+import {DocsCodeToken} from '../extensions/docs-code/docs-code';
+import {chromium} from 'playwright-core';
```

Comment:
Why aren't we using Pupeteer?

Reply:
I couldn't get puppeteer to work, I could not figure out why

Reply:
We should try to get it work as we are introducing another dependency for this and we should consistent automation, if we already have webdriver + pupeteer in all repos

Reply:
I'm happy to help with this

Reply:
I think that for now I would like to unblock the work within adev on angular/angular that is blocked on getting this support in place.

We can replace this at any point as its happening under the hood anyway. As I tried and investigated it seemed mostly to be issues around the fact that its trying to create a bunch of tmp files/directories in read only workspaces.

Reply:
I did run it with Bazel in the past to perform screenshot diffing.

Can you create an issue for tracking? We should definitely revisit what we use for automation, so that we use the same thing consistently

Reply:
Done https://github.com/angular/dev-infra/issues/1784